polling_stations/apps/data_importers/management/commands/import_eoni_from_s3.py

```diff
@@ -196,6 +197,12 @@ def handle(self, *args, **options):
             # Run the import
             self.call_import_eoni_command(options)
 
+        except Exception as e:
```

Comment:
If you really want to catch generic exception, you at least need to exclude `KeyboardInterrupt` and `SystemExit`. The pattern I'd usually use for this is:

```py
try:
    # generic thing that might fail in an unknown way
except (KeyboardInterrupt, SystemExit):
    raise
except Exception as err:
    # exception behaviour
    # maybe re-raise
```

same for the other place